locales/de.yml

```diff
@@ -37,15 +37,16 @@ de:
     arabic: Arabisch
     aragonese: Aragonesisch
     armenian: Armenisch
+    artificial: Kunstsprache (andere)
     australian: Australische Sprachen
     basque: Baskisch
     bengali: Bengalisch
     bosnian: Bosnisch
     bulgarian: Bulgarisch
     catalan: Katalanisch
-    celtic_other: Celtic (Other)
+    celtic_other: Keltisch (andere)
     chinese: Chinesisch
-    creoles_pidgins_other: Creoles and Pidgins (Other)
+    creoles_pidgins_other: Creoles and Pidgins (andere)
```

Comment:
"and"? Should this be "und"?

Reply:
@MFalletta This is your call here

Reply:
DE: Kreolisch und Kauderwelsch


horrible languages

Reply:
Fixed!